### PR TITLE
add getPost which accepts hash and returns post

### PR DESCRIPTION
**Pull request by Connoropolous**
This is to go with this UI post that Robert Best, Haiz, and I built together in the react app
https://github.com/Connoropolous/clutter/commit/2e74e9ff794dab5f757e487cd2c02716047c42cd

### CODE REVIEW

**Comment by Connoropolous** [comment]
Resolves #22

File: dna/clutter/clutter.js [hunk 1]
```diff
@@ -58,6 +58,20 @@ function postMod(params) {
     return update("post",post,hash);
 }
 
+function getPost(params) {
```
Comment (zippy) [inline dna/clutter/clutter.js]: Before merging this I'd love for this function to have a test added to [tests/posts.json](https://github.com/Holochain/clutter/blob/master/test/posts.json)  Would you like to add that, or should I?

**Comment by Connoropolous** [comment]
I can add

**Comment by Connoropolous** [comment]
@zippy I added that test now :)

**Comment by Connoropolous** [comment]
I also changed the shape of the post response back to matching the other responses. I really don't like the shape of the response, but it's probably best just to have them all match then to have one that's different. I also returned the HC error if its not found, instead of undefined

**Comment by zippy** [comment]
I actually agree with you on the shape of the responses.  I kinda just hacked it together.  Happy to see it changed to something more rational... :-)

**Pull request merged**